README.md

```diff
@@ -19,6 +19,31 @@ To build enry's CLI you must run
 it generates a binary in the project's root directory called `enry`. You can move this binary to anywhere in your `PATH`.
 
 
+### Faster regexp engine
+
+[Onigumura](https://github.com/kkos/oniguruma) is CRuby's regular expression engine.
```

Comment:
Was confused reading this first time and had the same question as https://github.com/src-d/enry/pull/113#issuecomment-333099095

What do you think about highlighting here an optional nature of this improvement? 
Like`Optionally, you can enable [Origumara] - a CRuby's ..`